lms/resources/_js_config/file_picker_config.py

```diff
@@ -19,7 +19,7 @@ def d2l_config(cls, request, _application_instance):
             config["listFiles"] = {
                 "authUrl": request.route_url(D2L.route.oauth2_authorize),
                 "path": request.route_path(
-                    "d2l_api.courses.files.list",
+                    "api.courses.files.list",
```

Comment:
Use the generic view.

This a very tiny improvement here but with a few like these most of this file becomes unnecessary.